packages/mds-logger/index.ts

```diff
@@ -47,26 +69,32 @@ const log =
       return {}
     }
 
-    const { log_message, log_data } = { log_message: redact(message), log_data: redact(data) }
-    const log_timestamp = Date.now()
-    const log_ISO_timestamp = new Date(log_timestamp).toISOString()
-    const log_requestId = httpContext.get('x-request-id')
+    const formattedMessage = formatLogMessage(level, message, data)
+    const { log_message, log_data } = formattedMessage
 
-    logger[level]({
-      log_level: level.toUpperCase(),
-      log_ISO_timestamp,
-      log_timestamp,
-      ...(log_requestId ? { log_requestId } : {}),
-      log_message,
-      log_data
-    })
+    logger[level](formattedMessage)
 
     return { log_message, log_data }
   }
 
+/**
+ * Follows the standard pattern of the [debug](https://www.npmjs.com/package/debug) module,
+ * with the caveat that logging function supports the mds-logger format you know and love.
+ * Default namespace is mds, but if you provide a namespace arg it will appear as 'mds:${namespace}'.
+ */
+const debugLog = (namespace?: string) => (message: string, data?: Record<string, unknown> | Error) => {
+  const formattedLogMessage = formatLogMessage('debug', message, data)
+  const { log_message, log_data } = formattedLogMessage
+
+  // Invoke the debug module with custom namespace & formatted message
+  debug(namespace ? `mds:${namespace}` : 'mds')(formattedLogMessage)
+
+  return { log_message, log_data }
+}
```

Comment:
I think `debug()` only needs to be invoked one time.

```suggestion
const debugLog = (namespace?: string) => {
  const debugLogger = debug(namespace ? `mds:${namespace}` : 'mds')
  return (message: string, data?: Record<string, unknown> | Error) => {
    const formattedLogMessage = formatLogMessage('debug', message, data)
    const { log_message, log_data } = formattedLogMessage
    // Invoke the debug module with custom namespace & formatted message
    debugLogger(formattedLogMessage)
    return { log_message, log_data }
  }
}
```

Reply:
Fixed in a581f89!